bedrock/press/templates/press/includes/resources.html

```diff
@@ -1,12 +1,8 @@
-<aside class="sidebar">
-  <h2>{{ _('Resources') }}</h2>
-  <ul>
-    <li><a href="https://blog.mozilla.org/press/ataglance/">{{ _('Mozilla at a Glance') }}</a></li>
-    <li><a href="https://blog.mozilla.org/press/media-library/">{{ _('Media Library') }}</a></li>
-    <li><a href="https://blog.mozilla.org/press/awards/">{{ _('Awards') }}</a></li>
-    <li><a href="https://blog.mozilla.org/press/kits/">{{ _('Firefox Press Kits') }}</a></li>
-    <li><a href="https://blog.mozilla.org/press/bios/">{{ _('Speaker Bios') }}</a></li>
-    <li><a href="{{ url('press.press-inquiry') }}">{{ _('Press Inquiry') }}</a></li>
-    <li><a href="{{ url('press.speaker-request') }}">{{ _('Speaker Request Form') }}</a></li>
-  </ul>
-</aside>
+<h4 class="mzp-c-sidemenu-title">{{ _('Resources') }}</h4>
```

Comment:
The side menu doesn't seem to quite have the correct markup for the styles to work: https://protocol.mozilla.org/patterns/organisms/sidebar-menu.html

See for example https://www.mozilla.org/en-US/security/

Reply:
Should I be using `sidemenu_lists()`?

Reply:
Up to you, but that would be one way to make sure the markup is as expected